allTimeEvents.php

```diff
@@ -0,0 +1,74 @@
+<?php
+require "logged_in_check.php";
+require "database_connect.php";
+require "set_session_vars_short.php";
+$pageTitle = "All Time Events";
+?>
+
+<?php
+
+$query = $db->prepare("SELECT memberID FROM Member");
```

Comment:
This PHP logic seems like a lot of work to do something that could be done with one SQL query that looks something like:

```sql
SELECT m.memberID, m.firstName, m.lastName,
       eventCount = (SELECT COUNT(att.id)
                         FROM AttendsEvent att
                        WHERE att.memberID = m.memberID)
FROM Member m
```

Reply:
Yes, the RRC DB is pretty small but you're hitting it 2 times per N members and the DB is nearly 10 years old at this point, so...

Reply:
You may consider retrieving and batching the memberIDs to optimize this a bit, but that's probably unnecessary given the traffic to the page.